src/main/scala/mesosphere/marathon/core/leadership/LeadershipModule.scala

```diff
@@ -0,0 +1,46 @@
+package mesosphere.marathon.core.leadership
+
+import akka.actor.{ ActorRef, ActorRefFactory, Props }
+import mesosphere.marathon.core.leadership.impl.{
+  LeadershipCoordinatorDelegate,
+  LeadershipCoordinatorActor,
+  WhenLeaderActor
+}
+
+class LeadershipModule(actorRefFactory: ActorRefFactory) {
```

Comment:
Please add some documentation for this module as well the provided objects/functionality.